examples/complete/versions.tf

```diff
@@ -0,0 +1,16 @@
+terraform {
```

Comment:
I'm unsure why this file cannot be read

Reply:
ok, will do

Reply:
I run `make pr/auto-format` successfully on my local environment